packages/traceability/issuer-api/src/pods/certification-request/certification-request.controller.ts

```diff
@@ -122,7 +122,7 @@ export class CertificationRequestController {
             new ValidateCertificationRequestCommand(dto)
         );
 
-        if (!validationCheck.success) {
+        if (!validationCheck?.success) {
             return validationCheck;
```

Comment:
This code doesn't make sense now - if validationCheck can be undefined, and THEN we enter the if, the undefined will be returned

Reply:
Good point, didn't pay too much attention to this just wanted it to compile. Will rework this part.

Reply:
Undid the change for now, as it's not needed it seems. Probably pushed by mistake.